rdrf/rdrf/settings.py

```diff
@@ -640,3 +641,7 @@
     'robots_txt',
     'js_reverse',
 )
+
+# URLs whitelisted for meeting the security conventions
+# Refer to docs/security/README.rst
+SECURITY_WHITELISTED_URLS = list(SECURITY_WHITELISTED_URLS)
```

Comment:
- Why do you need the `list`?
- I think it would read better if you used `from rdrf.security import url_whitelist` and made this line `SECURITY_WHITELISTED_URL = url_whitelist.SECURITY_WHITELISTED_URL`.

Reply:
I was trying to ensure that developers wouldn't modify the list in `url_whitelist` in angelman or mnd (in the code), but only modify the list in `settings`. But maybe a better way would be to make `url_whitelist.SECURITY_WHITELISTED_URLS` a tuple?

Didn't realise it'd work with `from rdrf.security import url_whitelist ` :+1:. I tried using `rdrf.security.url_whitelist` because `rdrf` is already imported.

Reply:
Yes, if that is your intent, using a tuple makes the most sense.
It will tell the reader immediately that you want a read-only list.

Reply:
BTW, you need associated tasks to do the changes in all the projects that have custom views like mnd and angelman (at least).